app/build.gradle

```diff
@@ -71,22 +71,23 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    namespace 'dev.arkbuilders.rate'
 }
 
 dependencies {
-    implementation "androidx.compose.ui:ui:1.4.3"
-    implementation "androidx.navigation:navigation-compose:2.5.2"
-    implementation "androidx.compose.material:material:1.4.3"
-    implementation "androidx.compose.ui:ui-tooling-preview:1.4.3"
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.5.1'
-    implementation 'androidx.activity:activity-compose:1.5.1'
+    implementation "androidx.compose.ui:ui:1.5.4"
+    implementation "androidx.navigation:navigation-compose:2.7.5"
+    implementation "androidx.compose.material:material:1.5.4"
+    implementation "androidx.compose.ui:ui-tooling-preview:1.5.4"
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.6.2'
+    implementation 'androidx.activity:activity-compose:1.8.1'
 
-    implementation "com.google.dagger:dagger:2.49"
-    kapt "com.google.dagger:dagger-compiler:2.49"
+    implementation "com.google.dagger:dagger:2.48"
```

Comment:
@MohamedPresident So we're degrading `com.google.dagger:dagger` version?

Reply:
Yeah for ksp Dagger still recommend  2.48